src/components/date-picker/DatePicker.tsx

```diff
@@ -399,6 +524,50 @@ class DatePicker extends React.Component<DatePickerProps> {
         this.onSelectHandler(null);
     };
 
+    renderCalendarButton = () => {
+        const { intl, isAccessible, isAlwaysVisible, isDisabled } = this.props;
+        const { formatMessage } = intl;
+
+        if (isAlwaysVisible) {
+            return null;
+        }
+
+        // De-emphasizing the Pikaday date picker because it does not meet accessibility standards
+        // Screenreaders & navigating via keyboard will no longer pick up on this element
+        if (isAccessible) {
```

Comment:
Could we make use of `aria-hidden="true"` or something instead to remove it from being read by screenreaders?

Reply:
I'll look into this

Reply:
I've updated this - I also set tabindex=-1 because the button should not be navigable by keyboard: https://github.com/box/box-ui-elements/pull/2751/files#diff-8c22974687f38296f0e4c9b61ba406c1a06dd3be54e61892b6a9ad5f546d0478R540